Loading rhel7/default-ccl
  Loading requirement: dot slurm turbovnc/2.0.1 vgl/2.5.1/64 singularity/current
    rhel7/global intel/compilers/2020.2 intel/mkl/2020.2 intel/impi/2020.2/intel
    intel/libs/idb/2020.2 intel/libs/tbb/2020.2 intel/libs/ipp/2020.2
    intel/libs/daal/2020.2 intel/bundles/complib/2020.2 cmake/latest
Changed directory to /rds/user/mrn31/hpc-work/euler/runs/scaling_runs.

JobID: 21820177
======
Time: Tue  6 Jun 12:58:20 BST 2023
Running on master node: cpu-q-367
Current directory: /rds/user/mrn31/hpc-work/euler/runs/scaling_runs
/usr/bin/perl: symbol lookup error: /usr/local/software/slurm/slurm-22.05.9/lib64/perl5/auto/Slurm/Slurm.so: undefined symbol: Perl_xs_apiversion_bootcheck

Nodes allocated:
================


numtasks=76, numnodes=1, mpi_tasks_per_node=76 (OMP_NUM_THREADS=1)

Executing command:
==================
mpirun -ppn 76 -np 76  

/home/mrn31/.conda/envs/kwant/lib/python3.8/site-packages/kwant/solvers/default.py:16: RuntimeWarning: MUMPS is not available, SciPy built-in solver will be used as a fallback. Performance can be very poor in this case.
  warnings.warn("MUMPS is not available, "
Traceback (most recent call last):
  File "./par_Kagome_Gauss_L150_p3.py", line 408, in <module>
    main()
  File "./par_Kagome_Gauss_L150_p3.py", line 399, in main
    for result in results:
  File "/home/mrn31/.conda/envs/kwant/lib/python3.8/concurrent/futures/process.py", line 484, in _chain_from_iterable_of_lists
    for element in iterable:
  File "/home/mrn31/.conda/envs/kwant/lib/python3.8/concurrent/futures/_base.py", line 611, in result_iterator
    yield fs.pop().result()
  File "/home/mrn31/.conda/envs/kwant/lib/python3.8/concurrent/futures/_base.py", line 439, in result
    return self.__get_result()
  File "/home/mrn31/.conda/envs/kwant/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
concurrent.futures.process.BrokenProcessPool: A process in the process pool was terminated abruptly while the future was running or pending.
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=21820177.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
